CITATION.cff

```diff
@@ -11,9 +11,10 @@ authors:
     given-names: Mikael
   - family-names: Magnusson
     given-names: Måns
-    orcid: "https://https://orcid.org/0000-0002-0001-1047"
+    orcid: "https://orcid.org/0000-0002-0001-1047"
```

Comment:
Mis-clicked to this file, but there were things to fix and add.. Does not need testing. :)